### PR TITLE
Exposing data with confidence in PrestaShop 1.7

### PR DESCRIPTION
**Pull request by eternoendless**
-- Do not merge immediately --

### CODE REVIEW

File: news/_posts/2018-03-15-exposing-data-with-confidence.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    
+    // get list of filters applied to the cart object
+    $cartFilters = $filters['cart']->getFilters();
+    
+    // remove "price" from the product whitelist
+    $cartFilters['products']->removeFromWhitelist('price');
+}
+```
+
+##### Bonus: adding arbitrary data to the prestashop object
```
Comment (mickaelandrieu) [inline news/_posts/2018-03-15-exposing-data-with-confidence.md]: how about: "Bonus: make new information available in Front Office"

Comment (eternoendless) [reply news/_posts/2018-03-15-exposing-data-with-confidence.md]: Why not!

**Comment by mickaelandrieu** [comment]
@eternoendless you probably noticed the build fails cause of Algolia key: is it something we can do here to fix it? Will the website be updated if we don't?

**Comment by Quetzacoalt91** [comment]
The algolia key is only available on "local" builds, not on the pull-requests. This error is normal.

**Comment by mickaelandrieu** [comment]
@Quetzacoalt91 can't we pass the Algolia key to the build (using Travis BO) to make the build pass?

**Comment by eternoendless** [comment]
Either way, we may move to Hugo (which is 10x as fast) so I don't think we should bother with that right now.

**Pull request merged**